Include/internal/pycore_bitutils.h

```diff
@@ -170,6 +170,49 @@ _Py_bit_length(unsigned long x)
 }
 
 
+// Same routine as above, for size_t, if that is bigger than unsigned long
+// This means bitscanreverse (as used above) is not going to work
+static inline int
+_Py_bit_length_size_t(size_t x)
```

Comment:
I would prefer to add a _Py_next_pow2() function which takes a size_t parameter. So we can use more efficient implementation when __builtin_clzl() and _BitScanReverse() are not available. Like:

```
static inline uint32_t next_pow2(uint32_t x)
{
	x -= 1;
	x |= (x >> 1);
	x |= (x >> 2);
	x |= (x >> 4);
	x |= (x >> 8);
	x |= (x >> 16);
	return x + 1;
}
```

I suggest to require the argument to be ``>= 2``.

Reply:
This code is reference at http://graphics.stanford.edu/~seander/bithacks.html#RoundUpPowerOf2 for example.